flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -78,6 +82,7 @@ import org.apache.flink.util.{InstantiationUtil, NetUtils}
 import org.jboss.netty.channel.ChannelException
 
 import scala.annotation.tailrec
+import scala.collection.JavaConversions._
```

Comment:
The usage of implicit conversion via `JavaConversions` is discouraged. Better use `JavaConverters'` explicit conversion methods which you call via `.asScala`